src/modules/pos/validator/components/ValidatorsMonitorView/Validators.js

```diff
@@ -78,6 +80,30 @@ const ValidatorsMonitor = ({ watchList }) => {
     config: { params: { address } },
   });
   const isValidator = validators?.data?.length > 0 && !isLoadingValidators;
+  const {
+    appEvents: {
+      transactions: { rewards },
+    },
+  } = useContext(ApplicationBootstrapContext);
+  const notification = rewards.length;
```

Comment:
`rewards.length && BigInt(rewards[0]?.reward || 0) > BigInt(0)` check for reward to claim is > 0

Reply:
Fixed